### DISCUSSION TITLE
[Question]: openai Winding down the ChatGPT plugins beta

### DISCUSSION THREAD

**Discussion by pxz2016** [danny-avila/LibreChat#1877] [Q&A]
### What is your question?

https://help.openai.com/en/articles/8988022-winding-down-the-chatgpt-plugins-beta

### More Details

Thank you for using ChatGPT plugins. We’ve taken feedback from other users like you and used it to create [GPTs](https://openai.com/blog/introducing-gpts). Based on the adoption of GPTs by both users and builders, we’ve decided to wind down the plugin beta. 
 
The creation of new conversations with plugins will be disabled on March 19, 2024. Users will be able to continue existing plugin conversations until April 9, 2024.
 
During the plugins beta, the store had a little more than 1,000 plugins available. The [GPT Store](https://chat.openai.com/gpts) already has hundreds of thousands of GPTs spread across categories like writing, productivity, programming, education and more. We hope you are as excited for the future of GPTs as we are. This is just the start and we have much more to come!
 
When will the plugin store stop being accessible? 

A: On March 19, 2024, you will no longer be able to install new plugins or create new conversations with existing plugins.
 
What will happen to the conversations I had with different plugins? 

A: On March 19, 2024, we will stop the creation of new conversations with plugins but you will be able to continue existing conversations until April 9, 2024.

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by pxz2016** [comment]
What is the impact on gptplugin endpoint

**Comment by danny-avila** [comment]
No impact because LibreChat is not connected to ChatGPT.

Eventually, the Plugins endpoint will involve and expand to all endpoints and not just OpenAI.
